src/mainwindow.cpp

```diff
@@ -346,6 +342,12 @@ void MainWindow::setupMainWindow()
     m_textEdit = ui->textEdit;
     m_editorDateLabel = ui->editorDateLabel;
     m_splitter = ui->splitter;
+    m_foldersWidget = m_splitter->widget(0);
+    m_noteListWidget = m_splitter->widget(1);
```

Comment:
I had another look at the `mainwindow.ui`. Actually, these pointers are already available as `ui->frameLeft` and `ui->frameMiddle`. I would suggest we just use those.

![image](https://user-images.githubusercontent.com/531764/221967720-3e898f03-7007-4f3b-8438-25ce37f7feea.png)